classes/blocks/class-base-block.php

```diff
@@ -37,7 +37,11 @@ public function render( $attributes ) {
 
 		$data = $this->prepare_data( $attributes );
 
-		\Timber::$locations = P4GBKS_PLUGIN_DIR . '/templates/blocks';
+		$template_path = '/templates/blocks';
+		if ( 'enform' === static::BLOCK_NAME ) {
+			$template_path = '/templates/blocks/enform';
```

Comment:
Not an exception, I thought its good to group the enform block related template files in one folder. do you think we should keep them at same level of other blocks template files?

Reply:
Ah I see, this is needed because `Base_Block` uses the `const BLOCK_NAME` to load the template. But if we move only the main `enblock.twig` file outside of the `enform` directory, we can keep all other enform templates grouped inside of that directory and we wouldn't need this block specific code here.